app/models/client.rb

```diff
@@ -1,3 +1,7 @@
 class Client < ApplicationRecord
   has_many :raids, through: :raids_clients
+
+  validates :name, :address, :gender, :country_of_origin,
+    :started_us_residence_at, :family_in_us, :contact_phone_number,
```

Comment:
probably don't want to validate everything here - def. not validating  `gender`, `family_in_us`, `started_us_residence`, `contact_phone_number` because there will be a lot of NULLs.  really might only want to validate `name` and i think i added that.